packages/tasks/src/model-libraries.ts

```diff
@@ -489,6 +489,14 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path_extension:"safetensors"`,
 	},
+	"seed-story": {
+		prettyLabel: "SEED-Story",
+		repoName: "SEED-Story",
+		repoUrl: "https://github.com/TencentARC/SEED-Story",
+		filter: false,
+		countDownloads: `path:"pytorch_model.bin" OR path:"tokenizer.model"`,
```

Comment:
Maybe this based on the open discussion

```suggestion
		countDownloads: `path:"cvlm_llama2_tokenizer/tokenizer.model"`,
```